pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
Not sure I see what you mean by a stretch -- it's literally the word used in the [documentation](http://perldoc.perl.org/perlrun.html):

> perl -p causes Perl to assume the following loop around your program (...)

Perhaps by stretch you mean using the last letter of loop? I actually think it's a perfectly good anchor, just like the start of a syllable in the middle of a word would.